core/domain/prod_validators.py

```diff
@@ -4627,6 +4627,8 @@ def _get_external_instance_custom_validation_functions(cls):
 class UserQueryModelValidator(base_model_validators.BaseUserModelValidator):
     """Class for validating UserQueryModels."""
 
+    FOUR_WEEKS = datetime.timedelta(weeks=4)
```

Comment:
Should we use one constant in both the places?

Reply:
Yes, but I think it is good to have a double check.

Reply:
But I'm expecting someone will change the constant in the corn file and this will get unnoticed and later in the future once we will run this validator it will start failing.

Reply:
Yeah, I guess I'm quite indifferent about it. Done.

Reply:
Just forgot to remove this constant, now done.